internal/catnames/data.go

```diff
@@ -0,0 +1,772 @@
+package catnames
+
+import (
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+func init() {
```

Comment:
This is wasteful. Why not do it once and fix the file itself?

Reply:
done